salt/utils/win_dacl.py

```diff
@@ -1170,17 +1173,17 @@ def get_owner(obj_name):
 
     except MemoryError:
         # Generic Memory Error (Windows Server 2003+)
-        owner_sid = 'S-1-1-0'
+        owner_sid = 'S-1-0-0'
 
     except pywintypes.error as exc:
         # Incorrect function error (Windows Server 2008+)
         if exc.winerror == 1 or exc.winerror == 50:
-            owner_sid = 'S-1-1-0'
+            owner_sid = 'S-1-0-0'
         else:
             raise CommandExecutionError(
                 'Failed to get owner: {0}'.format(exc.strerror))
 
-    return get_name(win32security.ConvertSidToStringSid(owner_sid))
+    return get_name(owner_sid)
```

Comment:
@twangboy I think I understand all of the changes except this one. Mind commenting on why this was required?

Reply:
The get_name function was modified to accept any type of input and return the name. So it already does a `win32security.ConvertSidToStringSid` check. No need to do it twice.